app/setup.sh

```diff
@@ -1,3 +1,5 @@
 export FLASK_APP="manager.py"
 export FLASK_ENV=development
-export FLASK_RUN_PORT=3000
\ No newline at end of file
+export FLASK_RUN_PORT=3000
+
+source email_config.sh
```

Comment:
This makes email_config.sh required, but it's also stripped out by the .gitignore. Should we provide an empty email_config by default, or add logic here to check for existence first?